tests/test_examples/test_runs_via_configs/test_via_configs.py

```diff
@@ -59,3 +63,99 @@ def test_train_with_categories(accelerator: str, devices: int) -> None:
 @pytest.mark.parametrize("accelerator, devices", accelerator_devices_pairs())
 def test_val(accelerator: str, devices: int) -> None:
     run("val_mock.py", accelerator, devices)
+
+
+@pytest.mark.parametrize("accelerator, devices", accelerator_devices_pairs())
+def test_inference_error_two_sources_provided(accelerator: str, devices: int) -> None:
```

Comment:
it's better to use pytest options. You can run the test, which passes if the function failed with a specific exception type and `match=...` specific message. Links: [1](https://www.depends-on-the-definition.com/test-error-messages-with-pytest/), [2](https://docs.pytest.org/en/7.1.x/how-to/assert.html)